sdv/constraints/tabular.py

```diff
@@ -824,14 +831,22 @@ def is_valid(self, table_data):
             pandas.Series:
                 Whether each row is valid.
         """
-        satisfy_low_bound = self._lt(
-            self._get_low_value(table_data), table_data[self.constraint_column]
+        low = self._get_low_value(table_data)
+        high = self._get_high_value(table_data)
+
+        satisfy_low_bound = np.logical_or(
+            self._lt(low, table_data[self.constraint_column]),
+            np.isnan(low),
         )
-        satisfy_high_bound = self._lt(
-            table_data[self.constraint_column], self._get_high_value(table_data)
+        satisfy_high_bound = np.logical_or(
+            self._lt(table_data[self.constraint_column], high),
+            np.isnan(high),
         )
 
-        return satisfy_low_bound & satisfy_high_bound
+        return np.logical_or(
```

Comment:
Pure curiosity: It seems like we are mixing the `np.logical_or` with `&`. Maybe we could use `np.logical_and` too, to make it uniform?

Reply:
Yes, that's a good point, I can change it to be consistent.